pypeit/spectrographs/keck_lris.py

```diff
@@ -997,7 +1026,10 @@ def config_specific_par(self, scifile, inp_par=None):
 
         # Wavelength calibrations
         if self.get_meta_value(scifile, 'dispname') == '400/8500':  # This is basically a reidentify
-            par['calibrations']['wavelengths']['reid_arxiv'] = 'keck_lris_red_400.fits'
+            if self.name == 'keck_lris_red_mark4':
```

Comment:
I'm not totally sure, but it seems to me that this should be done by creating a `config_specific_par` method in class `KeckLRISRMark4Spectrograph`

Reply:
yeah, I considered that.  But felt:

1.  We want all the non-wavelength items too
2.  It is a bit easier to have them all in one place.

When we add yet more `arxiv` solutions, I think this portion will
probably branch off between detectors at the get-go.

Reply:
I guess I also meant, since `config_specific_par` does not exist in `KeckLRISRMark4Spectrograph`, is par['calibrations']['wavelengths']['reid_arxiv'] = 'keck_lris_red_mark4_R400.fits' even read when `keck_lris_red_mark4` is used as spectrograph?